t/api/04-jobs.t

```diff
@@ -891,6 +891,7 @@ subtest 'Expand specified Machine, Testsuite, Product variables' => sub {
             'PUBLISH_HDD_1'       => 'SLES-15-SP1-x86_64-1234@64bit-minimal_with_sdk1234_installed.qcow2',
             'ANOTHER_JOB'         => 'SLES-15-SP1-x86_64-1234@64bit-minimal_with_sdk1234_installed.qcow2',
             'HDD_1'               => 'SLES-15-SP1-x86_64-1234@64bit-minimal_with_sdk1234_installed.qcow2',
+            'BACKEND'             => 'qemu',
```

Comment:
Thinking out loud... is this revealing a bug? Or should we have another test case to account for this? :thinking:

Reply:
I think so.  This test also checks this setting after adding the `BACKEDN => 'qemu'`? Do we need to add a single test case for this?

Reply:
I think not. Because what you do is to harmonize the settings for job post and isos post. So if no other tests are failing I think this is a good sign